instrumentation/oppia_instrumentation_test.bzl

```
@@ -0,0 +1,40 @@
+"""
+Instrumentation macros to setup up end-to-end tests.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+def oppia_instrumentation_test(
+        name,
+        srcs,
+        deps):
+    """
+    Creates library, binary and a instrumentation test for each test suite.
+
+    Args:
+        name: str. The class name of the Test suite or the test file.
```

```suggestion
        name: str. The class name of the test suite.
```

Better to be specific.

---

Done